app/src/main/java/com/github/wnder/NavigationActivity.java

```diff
@@ -0,0 +1,68 @@
+package com.github.wnder;
+
+import android.os.Bundle;
+import android.view.View;
```

Comment:
Unused

Reply:
those 3 were used, but removed other ones in the same class that weren't used

Reply:
removed